### PR TITLE
Japanese - Tips for Developing

### PR DESCRIPTION
**Pull request by jaykishigami**

### CODE REVIEW

File: _tips/developing.ja.md [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+* **ユーザーストーリー**
+  * [視覚障害者にとっての代替テキストの価値を説明](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## ページの言語と言語の変更を特定する
+
+例えば `<html lang="en">` のように　`html`　タグで `lang` . Use the `lang` 属性を使うことで､すべてのページの主要言語を示します｡要素の言語がページの他の部分と異なる場合､特定の要素でlang属性を使います｡
```
Comment (remibetin) [inline _tips/developing.ja.md:135]: @jaykishigami "Use the" needs translation

File: _tips/developing.ja.md [hunk 2]
```diff
@@ -0,0 +1,642 @@
...
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: WAI-ARIAを使ったメニューの機能と状態の識別" class="example" %}
+
+<div class="non-standard">
+  <figure>
+      <div class="code">
+{:/}
+
+~~~html
+<nav aria-label="Main Navigation" role="navigation">
```
Comment (remibetin) [inline _tips/developing.ja.md:479]: @jaykishigami The `aria-label` needs translation

File: _tips/developing.ja.md [hunk 3]
```diff
@@ -0,0 +1,642 @@
...
+* **Tutorial**
+  * [ラベルの制御](/tutorials/forms/labels/)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## 画像の代替テキストを含める
+
+すべての情報画像と機能画像に画像の代替テキストが追加されていることを確認すること｡装飾的な画像には空の代替テキスト､`alt=""`を使うか､代わりにCSSに含めてください｡代替テキストは通常､文章コンテンツの責任者が提供します｡
```
Comment (remibetin) [inline _tips/developing.ja.md:113]: @jaykishigami Can you confirm to me the translation is equivalent to "Use empty alternative text, alt="" for decorative images, or include _[the decorative images]_ in the CSS instead"

I want to make sure the translation does not imply we recommend including the alternative text in the CSS.

File: _tips/developing.ja.md [hunk 4]
```diff
@@ -0,0 +1,642 @@
...
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## 可能な限りCAPTCHAを避ける
+
+CAPTCHAは多くの人々に問題を引き起こす｡ユーザー入力が人間によって生成されたものであることを検証するには､自動検出やインターフェイスのインタラクションなど､もっと使いやすい手段があります｡CAPTCHAを本当に含める必要がある場合は､理解しやすく､障害を持つユーザーのための代替手段を含むようにしてください｡下記にその例を示します｡
+
+* CAPTCHAを解く2つ以上の方法を提供すること｡
```
Comment (remibetin) [inline _tips/developing.ja.md:606]: @jaykishigami The original text is "Providing more than two ways to solve the CAPTCHAs" (i.e at least 3) and the translation seems to be "Providing two ways or more ways to solve the CAPTCHAs" or "more than one way".

File: _tips/developing.ja.md [hunk 5]
```diff
@@ -0,0 +1,642 @@
...
+* CAPTCHAを迂回できる人間の担当者へのアクセスを提供すること｡
+* ユーザー認証にCAPTCHAを要求しないこと｡
```
Comment (remibetin) [inline _tips/developing.ja.md:608]: @jaykishigami Is this translation about "not requiring CAPTCHAs for authorized users" or about "not requiring CAPTCHAs for user authentication"?

The original text is "Not requiring CAPTCHAs for **authorized users.**"

**Review by remibetin — COMMENTED** [review]
Thank you @jaykishigami for this translation!

I have fixed some code issues & added some comments for your consideration.
Can you have a look at these and apply changes where needed?

File: _tips/developing.ja.md (hunk 5 above)
Comment (jaykishigami) [reply _tips/developing.ja.md:608]: * 認証されたユーザーにはCAPTCHAを要求しないこと｡

File: _tips/developing.ja.md (hunk 1 above)
Comment (jaykishigami) [reply _tips/developing.ja.md:132]: 例えば `<html lang="en">` のように　`html`　タグで `lang` 属性を使うことで､すべてのページの主要言語を示します｡

File: _tips/developing.ja.md (hunk 2 above)
Comment (jaykishigami) [reply _tips/developing.ja.md:479]: エリアラベル

Comment (remibetin) [reply _tips/developing.ja.md:479]: @jaykishigami Sorry for the misunderstanding. I meant the `aria-label` value, i.e. "Main Navigation"

**Comment by remibetin** [comment]
Thank you @jaykishigami I have updated the translation according to your comments. 

There is only one comment left: https://github.com/w3c/wai-quick-start/pull/401#discussion_r1566089148

File: _tips/developing.ja.md (hunk 2 above)
Comment (jaykishigami) [reply _tips/developing.ja.md:479]: I see. "主たるナビゲーション"

**Comment by shawna-slh** [comment]
Thank you @jaykishigami !

**Comment by remibetin** [comment]
QA done: ✅

**Pull request merged**